google-cloud-spanner/src/test/java/com/google/cloud/spanner/connection/it/ITTransactionModeTest.java

```diff
@@ -142,8 +140,12 @@ public void testDoNotAllowBufferedWriteInReadOnlyTransaction() {
     try (ITConnection connection = createConnection()) {
       connection.execute(Statement.of("SET TRANSACTION READ ONLY"));
       assertThat(connection.isAutocommit(), is(false));
-      exception.expect(SpannerExceptionMatcher.matchCode(ErrorCode.FAILED_PRECONDITION));
-      connection.bufferedWrite(Mutation.newInsertBuilder("FOO").set("ID").to(1L).build());
+      try {
+        connection.bufferedWrite(Mutation.newInsertBuilder("FOO").set("ID").to(1L).build());
+        fail("");
```

Comment:
nit: Here and below, add an "Expected exception" message.

Reply:
Done